packages/backup-restore-operator/charts/templates/deployment.yaml

```diff
@@ -38,16 +38,19 @@ spec:
         - name: pv-storage
           persistentVolumeClaim:
             claimName: {{ include "backupRestore.pvcName" . }}
-          {{- end }}
-      {{- with .Values.nodeSelector }}
+          {{- end }}      
       nodeSelector:
+{{- include "windows-node-selector" . | nindent 8 }}
+{{- include "linux-node-selector" . | nindent 8 }}
```

Comment:
I thought you could only do one nodeSelector in a pod spec. Are you trying to allow this to go on either type of node ?

Reply:
yeah even with both nodeSelectors the chart was able to run the pod on a linux cluster node. Although after it was created the pod only had the `kubernetes.io/os: linux` nodeSelector. So looks like with both selectors specified it allows the pod to go on either type of node. But I haven't tested it for a windows node, so I removed that selector

Reply:
I think it just overwrites with whatever was set last. If you want to test you could flip them and try to deploy to linux and see if it fails.

Reply:
Does this need to run on windows for 2.5.0 ? Might be able to solve this with nodeAffinity, we can talk about it.

Reply:
based on offline discussion, we don't need to add it in for windows right now